core/src/test/scala/integration/kafka/api/ZkSaslSslConsumerTest.scala

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package kafka.api
+
+import kafka.server.KafkaConfig
+
+class ZkSaslSslConsumerTest extends SaslSslConsumerTest {
```

Comment:
One alternative to having this test would be to simply enable ZK in `SaslSslConsumerTest`. This would keep our test times lower and would give us the same coverage (although if that test fails, it may be a little harder to debug, but I think that's OK given the trade-off).